lib/restbase.js

```diff
@@ -164,6 +165,9 @@ RESTBase.prototype.defaultListingHandler = function(value, restbase, req) {
 
 // Special handling for external web requests
 RESTBase.prototype.defaultWebRequestHandler = function(req) {
+    // enforce the usage of UA
+    req.headers = req.headers || {};
+    req.headers['User-Agent'] = req.headers['User-Agent'] || this.rb_config.user_agent;
```

Comment:
> Unless there is a compelling reason to lower-case them, I think we should honor the standard (even if it means opening a separate issue to fix existing lower-cased headers).

I agree. We have a precedent too - [PR 214](https://github.com/wikimedia/restbase/pull/214) which introduced the `x-request-id` header (note the lowercasing).

The options seem to be:
1. Merge this as-is and take care of the rest in a follow up
2. Lower-case `User-Agent` (to `user-agent`) here and then fix everything in a follow-up
3. Just lower-case the UA header and keep the others lowercased as well

My vote goes to option (1)